src/main/java/org/geysermc/packconverter/api/utils/ImageUtils.java

```diff
@@ -284,7 +283,7 @@ public static boolean isEmptyArea(BufferedImage img, int subX, int subY, int wid
         BufferedImage subImage = img.getSubimage(subX, subY, width, height);
         for (int x = 0; x < subImage.getWidth(); x++) {
             for (int y = 0; y < subImage.getHeight(); y++) {
-                if (!(new Color(subImage.getRGB(x, y), true).equals(Color.TRANSLUCENT))) {
+                if (new Color(subImage.getRGB(x, y), true).getTransparency() != Color.TRANSLUCENT) {
```

Comment:
Don't think this will have the same effect, needs checking.

Reply:
In fact, the old `equals` check will never be `true` (and thus, its negation will never be `false`) as `new Color(subImage.getRGB(x, y), true)` is type `Color` and `Color.TRANSLUCENT` is type `int`.

I believe, this new check is the one you originally wanted to have there. If you don't want any new behavior, just remove the check entirely.